readme.md

```diff
@@ -221,6 +221,21 @@ queryString.parseUrl('https://foo.bar?foo=bar');
 //=> {url: 'https://foo.bar', query: {foo: 'bar'}}
 ```
 
+### .stringifyUrl(object, options?)
+
+The inverse of [`.parseUrl()`](https://github.com/sindresorhus/query-string#parseurlstring-options)
+
+The `object` are the same as result of `.parseUrl`
+
+The `options` are the same as for `.stringify()`.
+
+Returns a URL with query string.
+
+```js
+queryString.stringifyUrl({url: 'https://foo.bar', query: {foo: 'bar'}});
+// => 'https://foo.bar?foo=bar'
+```
```

Comment:
The text here should be in sync with the TS definition.

Reply:
@sindresorhus I'm not sure how to fix it. The line that you comment already sync with TS definition from this line
https://github.com/sindresorhus/query-string/pull/217/files#diff-b52768974e6bc0faccb7d4b75b162c99R213-R218

Reply:
I meant the whole addition, not just what's immediately above here.

> The inverse of [`.parseUrl()`](https://github.com/sindresorhus/query-string#parseurlstring-options)

^ is missing from the type definition.